plugins/api/src/_fns.js

```diff
@@ -1,3 +1,5 @@
+import makeHeaders from 'wtf_wikipedia/src/_fetch/_headers.js'
```

Comment:
Does this make sense as a way to re-use this code from the main wtf_wikipedia module? An alternative would be to pass it in to the plugin as part of the `models` object passed in to every plugin, but this felt like a utility class rather than a model. Advice appreciated.